src/restapi/fConfig.ts

```diff
@@ -39,7 +39,7 @@ export async function fSubmitOrUpdateZendeskConfigSubmit(req: Request, res: Resp
         const targetID = cValues.zd_target_id;
         const storeValues = call.values as AppConfigStore;
         storeValues.zd_target_id = targetID;
-        await configStore.storeConfigInfo(storeValues);
+        configStore.storeConfigInfo(storeValues);
```

Comment:
Why is this no longer awaited? That will cause issues with lambda that were solved with https://github.com/mattermost/mattermost-app-zendesk/pull/45. The lambda function will not wait for that http request to be finished because we aren't waiting for the promise to resolve here.

Reply:
@mickmister  - I think you are right. We'd need to await this as well, since otherwise the execution could continue and we run the risk that the lambda invoker returns before the offloaded `storeConfigInfo` wrapper executes. Though - it it prob. a smaller risk than before (when the `kvset` post was not awaited at all) - and that is prob. why we haven't seen regressions on the issues originally fixed by your  original PR - it is still a risk. @jfrerich, feel free to re-add the await prefix where I removed it.